### DISCUSSION TITLE
Automerge fails on GitHub merge queues when there is an in-queue failure

### DISCUSSION THREAD

**Discussion by Zemnmez** [renovatebot/renovate#25699] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovatebot/github-action@v39.1.1

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

1. When automerge is enabled, RenovateBot will set the PR to automerge.
2. When the PR is added to the merge queue and then fails to merge (for example with merge conflicts), GitHub removes the automerge status on the PR
3. RenovateBot does not re-add the automerge status to the PR, so a human must intervene and check it

Correct behaviour here would be to re-check automerge when RenovateBot updates the PR.

Example PR:
https://github.com/zemn-me/monorepo/pull/3951

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
This is not a problem/bug - it's expected and documented behavior.

You are welcome to request a feature though: "Re-attempt platform automerge after any git push"
